android/src/main/java/com/epicshaggy/biometric/AuthAcitivy.java

```diff
@@ -65,7 +71,10 @@ public void onAuthenticationSucceeded(@NonNull BiometricPrompt.AuthenticationRes
             @Override
             public void onAuthenticationFailed() {
                 super.onAuthenticationFailed();
-                finishActivity("failed");
+                currentRetries++;
+                if(currentRetries >= maxRetries) {
```

Comment:
Just a tiny remark and I might be wrong but since your attribute but is called max**Re**tries, this should probably be
`if(currentRetries > maxRetries) {`
Since otherwise the user only gets 5 tries, not 5 REtries

Reply:
Yes, you are absolutely right. Good finding 👌